tests/test_post_live_metrics.py

```diff
@@ -43,7 +142,8 @@ def test_post_live_metrics_skip_on_missing_token(caplog):
 
 def test_post_live_metrics_skip_on_schema_error(caplog, monkeypatch):
     monkeypatch.setenv(DVC_STUDIO_TOKEN, "FOO_TOKEN")
-    monkeypatch.setenv(STUDIO_REPO_URL, "FOO_REPO_URL")
+    monkeypatch.setenv(DVC_STUDIO_REPO_URL, "FOO_REPO_URL")
+    monkeypatch.setenv(DVC_STUDIO_URL, STUDIO_URL)
```

Comment:
Is this line needed?

Reply:
Without it, we log a debug message about the url, so either we need to include it or retrieve a different record from `caplog.records` below.

Reply:
Got it!